### DISCUSSION TITLE
onelink for mobil applications

### DISCUSSION THREAD

**Discussion by Sinitra** [YOURLS/YOURLS#3702] [Q&A] [answered]
Hi, Do you have any support or plugins for mobile application directions? I couldn't find any. There is a URL shortening service called **onelink**, which allows redirection to multiple markets with one URL (Apple AppStore, Google Play, Huawei AppGallery, Microsoft Store). Does YOURS have a feature like this? I've searched a lot but couldn't find it. Did I miss it?

Best regards

**Comment by dgw** [comment, marked as answer]
If you've not managed to find such a plugin, it probably doesn't exist yet.

A plugin built around user-agent detection ([simple, but old, example](https://stackoverflow.com/a/24990323/5991)) would probably work but would need to figure out how to store and retrieve the different redirection options. YOURLS only supports one URL per keyword.

**Comment by Sinitra** [comment]
@dgw thank you
